submissions/peterIyiola/html/productPage-1.html

```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```

Comment:
You are only required to add a contact form page, the other files added are not needed.